src/main/scala/is/hail/utils/richUtils/RichCodeMemoryBuffer.scala

```diff
@@ -79,6 +80,36 @@ class RichCodeMemoryBuffer(val region: Code[MemoryBuffer]) extends AnyVal {
     region.invoke[Long, Long, Boolean]("loadBit", byteOff, bitOff)
   }
 
+  def loadAnnotation(typ: Type): Code[Long] => Code[_] = typ match {
```

Comment:
Again, we shouldn't be calling a Code[_] an annotation.  I'd maybe just call this `load`?

Reply:
Or `loadPrimitive` to match the other PR?